### ISSUE TITLE
Change the name

### ISSUE THREAD

**Issue by robwormald** [angular/angular2-seed#26]
@mgechev's angular2-seed has been around a while, and frankly is better supported. our eventual goal is to have the CLI as the solution, but this repo will probably stick around for a while. Let's avoid naming collisions.

suggestions?

**Comment by NathanWalker** [comment]
`angular2-sample-seed`
`angular2-seed-example`

Neither are in use anywhere.

**Comment by Bigous** [comment]
Or better... we could vote for turning mgechev/angular2-seed, the "official" one... pointing this one to his or transfering his to here and giving them the autonomy about the repo ;)

**Comment by pkozlowski-opensource** [comment]
-1 for rename. IMO we need something "official" that we can point people to and keep up to data + following best practices

+1 for working with other seed authors to converge into one. But before doing so we need to answer one question: what is a goal of such seed or what should be showcased? Unfortunately most of the popular seeds I've seen so far are quite big and not really suitable for "getting off the ground" 

But yeh, we need to have one "minimal" and "official" (whatever it means) starter kit.

**Comment by joachimprinzbach** [comment]
I am also not a big fan of just renaming the project to avoid naming clashes. 
The goals and the targeted developers might be very different. 

As @pkozlowski-opensource mentioned - there are seed-projects that work kind as a quick start and there are also ones that might even be ready to ship a production ready version of your application. 

For me, something like the angular-cli-project would be a very nice solution. 
You can start with something like a basic, easy understandable setup for playing around. 
Later you can add testing-support, production-build and so on. Having a tool that is able to glue a couple of often used modules together would be really helpful.

So basically you could have one single seed repo with a minimal setup to get started. With the cli tool you can add modules based on your needs. You could even specify which build-tool to use etc.

**Comment by NathanWalker** [comment]
^ all the above +1. It's not a huge deal to rename, however keep in mind it would likely create some confusion to newcomers whom are trying to evaluate different seeds in making decisions on what makes sense for their project. (who's a fork?, which is which?, which one did our team like again?, angular's, mgechev, my brothers, my sisters `angular2-seed`)

Perhaps biggest help would be as @pkozlowski-opensource mentions, collab with other seeds to create the best for everyone, maybe a starter seed ("minimal"), a playground with examples ("fun learning"), and one for a large production app setup ("official").
